ui/apps/platform/src/utils/dateUtils.ts

```diff
@@ -1,49 +1,135 @@
-import { distanceInWordsStrict, format } from 'date-fns';
+import Raven from 'raven-js';
+import { distanceInWordsStrict } from 'date-fns';
 
-const timeFormat = 'h:mm:ssA';
-const dateFormat = 'MM/DD/YYYY';
-const dateTimeFormat = `${dateFormat} | ${timeFormat}`;
+export type DateLike = string | number | Date;
 
-type DateLike = string | number | Date;
+const defaultDateFormatOptions = {
```

Comment:
Help my Monday mind whether `Intl.DateTimeFormatOptions` for these constants (imitation function arguments below) provides any benefit for type checking?

Reply:
Initially I thought no, that the `as const` made this roughly equivalent. However since we only want `Intl.DateTimeFormatOptions` as properties of these objects, annotating with `: Readonly<Intl.DateTimeFormatOptions>` is a little better. That would prevent us from adding invalid properties down the line which would be ignored, and may slip through testing.